examples/evm/deposit-address/README.md

```diff
@@ -2,6 +2,17 @@
 
 This test demonstrates how to deposit tokens from one chain to another using a deposit address.
 
+
+### Context
+
+This example demonstrates how to deposit 10e6 tokens from Avalanche chain to Fantom chain using a deposit address. The initial balance of the Avalanche chain and Fantom chain is `100000000`. After the execution, the balance of Avalanche chain is `90000000` and the balance of Fantom chain is `109000000`. The deposit address is `0xb54eA64537F3307907E06d3B93ccd4A3E711623f`.
```

Comment:
It seems the stated initial balance of 100000000 is actually 1e18 when I log it.
I am pushing a change that logs 1.000 as our output from the script instead of 1000000000000000000 (as it currently is). And I'll update the README here to reflect that.